hare/broker_test.go

```diff
@@ -50,35 +50,21 @@ func createMessage(tb testing.TB, instanceID types.LayerID) []byte {
 	return mustEncode(tb, msg.Message)
 }
 
-func TestBroker_Start(t *testing.T) {
-	broker := buildBroker(t, t.Name())
-
-	err := broker.Start(context.Background())
-	assert.Nil(t, err)
-
-	err = broker.Start(context.Background())
-	assert.NotNil(t, err)
-	assert.Equal(t, "hare broker already started", err.Error())
-
-	closeBrokerAndWait(t, broker.Broker)
-}
-
 // test that a InnerMsg to a specific set ID is delivered by the broker.
 func TestBroker_Received(t *testing.T) {
 	broker := buildBroker(t, t.Name())
 	broker.mockSyncS.EXPECT().IsSynced(gomock.Any()).Return(true).AnyTimes()
 	broker.mockSyncS.EXPECT().IsBeaconSynced(gomock.Any()).Return(true).AnyTimes()
 	broker.mockStateQ.EXPECT().IsIdentityActiveOnConsensusView(gomock.Any(), gomock.Any(), gomock.Any()).Return(true, nil).AnyTimes()
-	require.NoError(t, broker.Start(context.Background()))
+	broker.Start(context.Background())
+	defer broker.Close()
```

Comment:
```suggestion
	t.Cleanup(broker.Close())
```

NIT: `defer` isn't executed when a panic happens, `t.Cleanup()` is.

Reply:
done.